lib/events.ts

```diff
@@ -78,34 +77,34 @@ export async function trackCommand(commandSignature: string) {
 			}
 			try {
 				const balena = getBalenaSdk();
-				const username = await balena.auth.whoami();
+				const $username = await balena.auth.whoami();
 				storage.set('cachedUsername', {
 					token,
-					username,
+					username: $username,
 				} as CachedUsername);
-				return username;
+				return $username;
 			} catch {
 				return;
 			}
-		})(),
-		mixpanel: getMixpanel(balenaUrl),
-	})
-		.then(({ username, mixpanel }) => {
-			Sentry.configureScope((scope) => {
-				scope.setUser({
-					id: username,
-					username,
-				});
-			});
-			return mixpanel.track(`[CLI] ${commandSignature}`, {
-				distinct_id: username,
-				version: packageJSON.version,
-				node: process.version,
-				arch: process.arch,
-				balenaUrl, // e.g. 'balena-cloud.com' or 'balena-staging.com'
-				platform: process.platform,
+		})();
+
+		const mixpanel = getMixpanel(balenaUrl);
+
+		Sentry.configureScope((scope) => {
+			scope.setUser({
+				id: username,
+				username,
 			});
-		})
-		.timeout(100)
```

Comment:
I didn't notice any change by removing this and the mixpanel checking code seems to take > 100ms for me already. I've got a feeling it didn't actually have any effect in reality because if the request is still in progress it'll keep nodejs running until completed even if we've timed out this promise